### ISSUE TITLE
AppImage is for Linux

### ISSUE THREAD

**Issue by ambitslix** [Conky-for-macOS/Manage-Conky#27]
The link gets you:

file conky-x86_64.AppImage 
conky-x86_64.AppImage: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.18, stripped

```
curl -sL -o conky-x86_64.AppImage \
    $(curl -sL https://api.github.com/repos/brndnmtthws/conky/releases/latest | \
    jq --raw-output '.assets[0] | .browser_download_url')
```

Expected an macos binary.

**Comment by npyl** [comment]
Hey
I think that AppImage is only available for Linux.

If you want to install conky on your macOS, please use one of the following ways:
- Build it yourself
- Install using homebrew:
```
brew cask install xquartz
brew tap Conky-for-macOS/homebrew-formulae
brew install conky
```

As for why I keep this command on the Readme, it is because `conky-for-macOS` is used mostly as a means for pushing macOS patches to upstream.  I do not intend to completely differenciate this fork from the original project.

**Comment by npyl** [comment]
I advice first trying with the second method; If this doesn't work, then you could try with the third or if you don't want ManageConky, tell me so I can customise the steps a bit to your liking.

Please be patient because this project is a bit difficult to maintain and sometimes it takes a bit for me to get on track with it after a long time 😃

**Comment by ambitslix** [comment]
I prefer building it but macports is having issues with freetype, if you know how to get past that let me know. I will try the other methods suggested. 

Thank you!

**Comment by npyl** [comment]
I have never tried out building conky with macports.
What error does it give you?

**Comment by ambitslix** [comment]
cmake ..
-- CMAKE_C_FLAGS: -std=c99
-- CMAKE_CXX_FLAGS: -std=c++17
-- CMAKE_C_FLAGS_DEBUG: -g
-- CMAKE_CXX_FLAGS_DEBUG: -g
-- CMAKE_C_FLAGS_MINSIZEREL: -Os -DNDEBUG
-- CMAKE_CXX_FLAGS_MINSIZEREL: -Os -DNDEBUG
-- CMAKE_C_FLAGS_RELEASE: -O3 -DNDEBUG
-- CMAKE_CXX_FLAGS_RELEASE: -O3 -DNDEBUG
-- CMAKE_C_FLAGS_RELWITHDEBINFO: -O2 -g -DNDEBUG
-- CMAKE_CXX_FLAGS_RELWITHDEBINFO: -O2 -g -DNDEBUG
-- CMAKE_BUILD_TYPE: RelWithDebInfo
-- Looking for clock_gettime
-- Looking for clock_gettime - found
CMake Error at cmake/ConkyPlatformChecks.cmake:350 (message):
  Unable to find freetype library
Call Stack (most recent call first):
  CMakeLists.txt:35 (include)


-- Configuring incomplete, errors occurred!

 sudo port install freetype
--->  Computing dependencies for freetype
--->  Cleaning freetype
--->  Scanning binaries for linking errors
--->  No broken files found.
--->  No broken ports found.

**Comment by npyl** [comment]
Thanks.
This seems like a problem with conky's cmake files and not macports.
Let me see if I can amend the problem.

**Comment by ambitslix** [comment]
I manage to build conky itself by fixing it's cmake using macports. Can I use this manager?

**Comment by npyl** [comment]
Hey! Glad you fixed it!
I think I've seen this problem before.
Does grabbing the latest (v0.9.9) from [here](https://github.com/Conky-for-macOS/Manage-Conky/issues/26) fix the problem?

**Comment by npyl** [comment]
Did you fix your problem?

**Comment by ambitslix** [comment]
Yes, built conky from source, by fixing ConkyPlatformChecks.cmake.

**Comment by npyl** [comment]
What about the ManageConky app? Did it work with the newest beta I gave you?

**Comment by ambitslix** [comment]
It runs nut it complains about XQuartz being installed. I have it installed from macports also, perhaps it wants another version.

**Comment by npyl** [comment]
I don't think the version is the problem.
Probably it is the installation directory because the only way I knew to check for Xquartz was checking for it in `/Applications`.

Where is Xquartz installed on your computer? If it is in another location, is it the standard location of installation for macports?

**Comment by ambitslix** [comment]
Sure, looks like macports installs in
/opt/local
